mapbox_vector_tile/decoder.py

```diff
@@ -34,11 +36,16 @@ def get_message(self, pbf_data, y_coord_down=False):
                     value = self.parse_value(val)
                     props[key] = value
 
-                geometry = self.parse_geometry(feature.geometry, feature.type, layer.extent, y_coord_down)
-                new_feature = {"geometry": geometry, "properties": props, "id": feature.id, "type": feature.type}
+                geometry = self.parse_geometry(geom=feature.geometry, ftype=feature.type, extent=layer.extent)
+                new_feature = {"geometry": geometry, "properties": props, "id": feature.id, "type": "Feature"}
```

Comment:
tangentially, we might want to also collect a list of the user facing changes and document them in the readme under an upgrading section.